packages/pwa-kit-create-app/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## v3.0.0-dev (May 11, 2023)
+## v3.0.0 (June 15, 2023)
 
+-   Package name changed to `@salesforce/pwa-kit-create-app`
 -   Add `--templateVersion` argument to allow template version selection when generating a project using a template that is hosted on NPM. [#1229](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1229)
 -   Add extensible project generation support. [#1205](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1205)
 
```

Comment:
Do we want to note that previous versions don't use the @salesforce namespace? 
The same comment applies to the rest of the public packages under @salesforce namespace CHANGELOG.md files.